src/shelters/ShelterList.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from 'react';
+import Shelter from './Shelter';
+import shelterApi from '.././services/shelterApi';
+import Button from '@material-ui/core/Button';
+import './ShelterList.css';
+
+class ShelterList extends Component {
+	constructor(props) {
+		super(props)
+		this.state = { shelters: [] };
+  }
+
+  componentDidMount = async () => {
+    const response = await shelterApi.getShelters();
+    this.setState({ shelters: response.shelters })
+  }
+
+	redirectToTarget = () => {
+	 this.props.history.push('/vouchers')
+ }
+
+  render() {
+		return (
+			<div className="shelter-list-container">
+					{ this.state.shelters.map((shelter, i) => {
+							return <Shelter key={i} shelter={shelter}/>
+						})
```

Comment:
Find way to make it so that rendering shelters does not break the app - currently if shelters array is empty then throws error